src/ajitSJune20/CountLetDigSpclChar.java

```diff
@@ -0,0 +1,36 @@
+//Count Digits,Letters, UpperCase characters, LowerCase characters, Special characters from predefined string.
+
+package ajitSJune20;
+
+public class CountLetDigSpclChar {
+	
+	void displayCount(String str) {
+		int upper = 0, lower = 0, digit = 0, special = 0, letter=0; 
+		
+		for(int index=0;index<str.length();index++) {
+			char ch=str.charAt(index);
+			if(Character.isDigit(ch))
+				digit++;
+			else if(Character.isUpperCase(ch))
+				upper++;
+			else if(Character.isLowerCase(ch))
+				lower++;
+			else if(Character.isLetter(ch))
```

Comment:
No you still not getting my point, I have said, without using letter variable and else if(Character.isLetter(ch)) block you can achieve the output. Still if you can't do it, will tell the solution.

Reply:
Made Changes. I hope its correct this time. 
![image](https://user-images.githubusercontent.com/66831507/85770998-20d3ed00-b739-11ea-9bde-0aa98517102b.png)